bin/i18n/utils/crowdin_client.rb

```diff
@@ -0,0 +1,221 @@
+require 'crowdin-api'
+
+require_relative '../i18n_script_utils'
+
+module I18n
+  module Utils
+    class CrowdinClient
+      RequestError = Class.new(StandardError)
+
+      CREDENTIALS_PATH = CDO.dir('bin/i18n/crowdin_credentials.yml').freeze
+      PROJECT_IDS = CDO.crowdin_project_ids.freeze
+      MAX_ITEMS_COUNT = Crowdin::Web::FetchAllExtensions::MAX_ITEMS_COUNT_PER_REQUEST.freeze
+      REQUEST_RETRY_ATTEMPTS = 2 # Number of retries for a failed request
+      REQUEST_RETRY_DELAY = 2 # Number of seconds to retry a failed request
```

Comment:
Nit: maybe something like,
```suggestion
      REQUEST_RETRY_DELAY = 2 # Number of seconds to wait before retrying a failed request
```